components/explore/ExploreSort.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div>
+    <o-dropdown
```

Comment:
`<NeoDropdown>`?

Reply:
+ 1 for neodropdown

Reply:
`@click.native="onChange"` on NeoDropdownItem maybe? but that doesn't look right multiple selection won't work
Or maybe use `NeoSelect` multiple instead

Reply:
I'm fine with it, I don't really see the point of multiple sorting options